### PR TITLE
Changes needed to run metal3-dev-env in the Ironic CI

### PR DESCRIPTION
**Pull request by dtantsur**
See commit messages for details.

### CODE REVIEW

File: 01_prepare_host.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ fi
 
 if [[ $OS == ubuntu ]]; then
   sudo apt-get update
-  sudo apt -y install python3-pip jq curl wget
+  sudo apt -y install python3-pip jq curl wget pkg-config
 
   # Set update-alternatives to python3
   if [[ ${DISTRO} == "ubuntu18" ]]; then
```
Comment (elfosardo) [inline 01_prepare_host.sh:19]: for ubuntu jammy we need to reload the correct kvm module based on brand (amd or intel), not just kvm_intel
I guess we could check cpuinfo for the virtualization feature, vmx on intel, svm for amd, and decide based on that

Comment (dtantsur) [reply 01_prepare_host.sh:19]: I agree, but we're also in a nested VM situation, and I think at least some VMs don't have hardware-assisted nested virtualization...

Comment (elfosardo) [reply 01_prepare_host.sh:19]: right, then we need to remove the reload in case we don't have kvm at all

**Comment by elfosardo** [comment]
probably need to rebase this as we changed k8s version

File: 01_prepare_host.sh (hunk 1 above)
Comment (elfosardo) [reply 01_prepare_host.sh:19]: this should avoid issues with nested virtualization or in general when we don't have kvm  https://github.com/dtantsur/metal3-dev-env/pull/1

**Comment by elfosardo** [comment]
I think we can close this, the changes merged already in https://github.com/metal3-io/metal3-dev-env/pull/1160

**Pull request closed without merging**